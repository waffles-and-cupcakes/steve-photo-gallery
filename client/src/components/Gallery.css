.banner-img {
  background: url("https://s3-us-west-1.amazonaws.com/wandcphotogallery/adorable-animal-canine-164446.jpg") no-repeat center top;
  background-position: center center;
  background-size: cover;
  height: 75vh;
  cursor: pointer;
}