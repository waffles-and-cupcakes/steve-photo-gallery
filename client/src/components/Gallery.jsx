import React from 'react';
import './Gallery.css';

const Gallery = () => (
  <div className="banner-img">
  </div>
);

export default Gallery;