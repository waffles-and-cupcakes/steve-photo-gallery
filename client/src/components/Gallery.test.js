import React from 'react';
import { shallow, mount, render, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Gallery from './Gallery.jsx';

configure({adapter: new Adapter() });

test('Gallery should render correctly', () => {
  const output = shallow(<Gallery />);
  expect(shallowToJson(output)).toMatchSnapshot();
});