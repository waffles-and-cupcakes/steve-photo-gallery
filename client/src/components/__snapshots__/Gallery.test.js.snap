// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery should render correctly 1`] = `
<div
  className="banner-img"
/>
`;
